api/swagger.yaml

```diff
@@ -177,6 +187,64 @@ paths:
           $ref: '#/components/responses/Unacceptable'
         '5XX':
           $ref: '#/components/responses/InternalServerError'
+          
+  /api/v1/login:
+    post:
+      tags:
+        - TeddyCloud
+      summary: Login
+      requestBody:
+        required: true
+        content:
+          "application/json":
```

Comment:
I would prefer using a POST request sending the data as parameters. Parsing JSON is not so easy in the backend, but just parsing the POST params is easy

Reply:
post params also referred to as url query params? 
Or just text/plain?

I'd not go for url query params for login requests because of in example they might occure in web server log files etc.

Reply:
Or multipart/form-data?
Guess multipart/form-data would be better than text/plain

Reply:
application/x-www-form-urlencoded
With the content of the parameters in the same format as query paramters in GET, but just as content of the response (url encoded)